app/shop.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+
+
+class Shop:
+    def __init__(
+            self,
+            name: str,
+            location: list[int],
+            products: dict
+    ) -> None:
+        self.name = name
+        self.location = location
+        self.products = products
+
+    def receipt(self, customer: object) -> str:
+        date = datetime.now()

```

Comment:
tests failed. To fix them do not use
from datetime import datetime
instead use
import datetime